grapher/core/GrapherConstants.ts

```diff
@@ -38,6 +38,7 @@ export enum StackMode {
 export const BASE_FONT_SIZE = 16
 
 export enum FacetStrategy {
+    together = "together", // No facets
```

Comment:
Is `none` maybe more appropriate? Since faceting is technically disabled.

Reply:
Agreed, fixed!